instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumViewTrackingE2ETests.kt

```diff
@@ -64,6 +65,7 @@ internal class RumViewTrackingE2ETests {
             )
         }
         launch(ViewTrackingActivity::class.java)
+            .moveToState(Lifecycle.State.DESTROYED)
```

Comment:
I wonder if this is only needed in these test suite 🤔

Reply:
Other tests have some UI interactions after `launch` call, so they have view updates before test is finished.